### PR TITLE
Add codespaces config for github remote dev

### PR DESCRIPTION
**Pull request by fuziontech**
# [Codespaces](https://github.com/features/codespaces) 🎉 
Since codespaces are free for open source development we should be using them for PostHog!

## Try it for yourself!
On this branch go up to `Open with` up in the top right of the PR and click `Open in codespace`
image

It does take a bit to start the entire stack (~5-10 minutes), but once it is started you are dropped into the web container shell.
To start posthog from web container run at the terminal in vscode:
```bash
./ee/bin/docker-ch-dev-backend
```
Give it a try!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,6 +69,7 @@
         "kea-localstorage": "^1.1.1",
         "kea-router": "^1.0.0",
         "kea-window-values": "^0.0.1",
+        "lodash": "^4.17.21",
```
Comment (fuziontech) [inline package.json:72]: @mariusandra - I have no idea if this is a good idea. This was blocking `yarn build` which was failing without it. Let me know if there is some other better fix here

**Comment by samwinslow** [comment]
Looks like this may not be available to all of us yet.

Screen Shot 2021-07-13 at 7 03 27 PM
Screen Shot 2021-07-13 at 7 04 06 PM

**Comment by fuziontech** [comment]
😞 I'm surprised by that! It's been around for a while and is quite stable now. Hopefully this goes GA sooner than later.

File: package.json (hunk 1 above)
Comment (mariusandra) [reply package.json:72]: Very strange. We don't use this library directly in our code. What was the error?

**Comment by mariusandra** [comment]
I have access, but some people I've asked to ask for access haven't gotten it yet (e.g. @piemets). So this is definitely useful and I might use it myself now, but its reach will be limited.

> To start posthog from web container run at the terminal in vscode:
> ./ee/bin/docker-ch-dev-backend

Any way this message could be `echo`ed inside the terminal?

And not sure regarding `lodash`. We don't use this library ourselves and I'd prefer to not have it in the list of things to maintain. What was the error?

File: package.json (hunk 1 above)
Comment (fuziontech) [reply package.json:72]: weird...I removed it and it's working fine now. 🤷‍♂️

**Comment by fuziontech** [comment]
done and done! Removing lodash seems to work fine now. Not sure what was going on there.

Also added MOTD that has the instructions in it/

**Comment by marcushyett-ph** [comment]
Oh... I'd love to use this - any ideas how to get in the Beta?

**Comment by neilkakkar** [comment]
https://github.com/features/codespaces/signup Good luck! xD

**Comment by marcushyett-ph** [comment]
image

Fingers very crossed.

**Pull request merged**